Mathlib/Combinatorics/SimpleGraph/Connectivity/WalkCounting.lean

```diff
@@ -240,6 +240,11 @@ lemma disjiUnion_supp_toFinset_eq_supp_toFinset {G' : SimpleGraph V} (h : G ≤
 
 end Fintype
 
+
+/-- The odd components are the connected components of odd cardinality. This definition excludes
+infinite components. -/
+abbrev oddComponents : Set G.ConnectedComponent := {c : G.ConnectedComponent | Odd c.supp.ncard}
+
 lemma ConnectedComponent.odd_card_supp_iff_odd_subcomponents [Finite V] {G'}
     (h : G ≤ G') (c' : ConnectedComponent G') :
     Odd c'.supp.ncard ↔
```

Comment:
I tried to modify this one as well, but I couldn't get the proof to work. @YaelDillies if you have any idea how approach this, I'm open to including it, but I'm also good keeping this one as-is.

Reply:
Done